backend/courses/management/commands/loadcourses.py

```diff
@@ -1,8 +1,13 @@
 import logging
```

Comment:
Do you think it also makes sense to have this check a runtime option so that it only runs during add/drop? That way we don't have to make any code changes to disable/enable it

Reply:
I was thinking about this but I think we want it to always run. We never know when people are going to be looking at courses, and it doesn't hurt to have the most up-to-date info.